packages/wonder-blocks-tokens/src/tokens/media-queries.ts

```diff
@@ -5,12 +5,18 @@
  *
  */
 
+/* Pure height values */
+const height = {
+    smMin: 500, // Arbitary breakpoint for screen height
+} as const;
+
 /* Pure width values */
 const width = {
     xsMax: 567,
     smMin: 568,
     smMax: 681,
     mdMin: 682,
+    mid: 767,
```

Comment:
I do not love the naming structure here. We sorta backed ourselves into a corner with `sm`, `md`, etc.! I'm open to suggestions. I moved this into tokens to make the removal of `MediaLayout` more backwards compatible and to deal with some "todo" comments in the code. It seemed more reasonable than trying to redesign a ton of breakpoint widths.